packages/manager/apps/web-ongoing-operations/src/data/api/document.ts

```diff
@@ -0,0 +1,67 @@
+import { apiClient } from '@ovh-ux/manager-core-api';
+import { OdsFile } from '@ovhcloud/ods-components';
+import axios from 'axios';
+
+export type MeDocument = {
+  id: string;
+  putUrl: string;
+};
+
+export type Tag = {
+  key: string;
+  value: string;
+};
+
+export type MeDocumentResponse = {
+  putUrl: string;
+  size: number;
+  id: string;
+  expirationDate?: string;
+  getUrl: string;
+  creationDate: string;
+  validationDate: string;
+  name: string;
+  tags: Tag[];
+};
+
+export const getMeDocument = async (id: string): Promise<MeDocument> => {
+  return apiClient.v6
+    .get(`/me/document/${id}`)
+    .then((res) => res.data as MeDocumentResponse);
+};
+
+const postMeDocument = async (
+  filename: string,
+): Promise<MeDocumentResponse> => {
+  return apiClient.v6
+    .post(`/me/document`, {
+      name: filename,
+    })
+    .then((res) => res.data as MeDocumentResponse);
+};
+
+const postMeDocumentCors = async (origin: string): Promise<void> => {
+  return apiClient.v6.post(`/me/document/cors`, {
+    origin,
+  });
+};
+
+const customAxiosInstance = axios.create({});
+
+const saveDocumentFile = async (
+  putUrl: string,
+  file: OdsFile,
+): Promise<void> => {
+  return customAxiosInstance.put(putUrl, file, {
+    headers: {
+      'Content-type': 'multipart/form-data',
+    },
+  });
+};
```

Comment:
You could apply same structure that on packages/manager/apps/web-ongoing-operations/src/data/api/ongoing-operations-actions.ts
Not async if no await and "v6" directly